src/Analyzer/FileVisitor.php

```diff
@@ -63,4 +63,9 @@ public function leaveNode(Node $node): void
             $this->classDescriptions[] = $classDescription;
         }
     }
+
+    private function isNewNodeType(Node $node): bool
```

Comment:
not sure i'm getting where this method is used. Apart from that, I think we should keep track why we are adding such checks and condition

Reply:
It was a refusal.